nf_core/modules/modules_json.py

```diff
@@ -274,7 +274,11 @@ def determine_module_branches_and_shas(self, install_dir, remote_url, modules):
                     found_sha = True
                     break
             if found_sha:
-                repo_entry[module] = {"branch": modules_repo.branch, "git_sha": correct_commit_sha}
+                repo_entry[module] = {
+                    "branch": modules_repo.branch,
+                    "git_sha": correct_commit_sha,
+                    "installed": "modules",
```

Comment:
why did you call the key `installed` instead of `installed_by`? `installed` makes me expect something like a boolean value.